src/app/garfile/amg/checkin/pane.njk

```diff
@@ -28,27 +29,54 @@
   <tr class="govuk-table__row">
     <td class="govuk-table__cell">
       <div>
-        <div class="status-disc {{statuscolour}}"></div>
+        <div class="status-disc {{statuscolour}}" title="{{statustitle}}"></div>
       </div>
     </td>
     <td class="govuk-table__cell">{{person.firstName}}
       {{person.lastName}}</td>
-    <td class="govuk-table__cell">
+    <td class="govuk-table__cell" style="position:relative;">
       {{statustitle}}
+      {% if code | length and(code === '1Z' or code | first === '0') %}
+        <div class="upt-hover">
+          {% if code === '1Z' %}
+          Pilots, operators and agents will receive a NO BOARD response via a call and email when Authority to Carry (ATC) has been refused.
+        {% elif(code === '0A') %}
+            The Home Office can find a valid permission to travel for the person. Pilots, operators and agents are not required to check visas, but Passport or Travel Document checks still apply. They must check that the passport or travel document presented is genuine and valid, and that the person is the rightful holder.
+            <br/><br/>For more information on checking passports and travel documents click here  <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1134497/Guidance_on_examining_identity_documents.pdf">Guidance on examining identity documents (publishing.service.gov.uk)</a>
+          {% else %}
+            Pilots, operators and agents are required to check visas for Visa Nationals. Passport or Travel Document checks apply for all passengers. They must check that the passport or travel document presented is genuine and valid, and that the person is the rightful holder.
+            <br/><br/>For more information on checking passports and travel documents click here <a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/1134497/Guidance_on_examining_identity_documents.pdf">Guidance on examining identity documents (publishing.service.gov.uk)</a>
+            Follow this link to check visa requirements <a href="https://www.gov.uk/check-uk-visa">Check if you need a UK visa - GOV.UK (www.gov.uk)</a>
+          {% endif %}
+        </div>
+      {% endif %}
     </td>
   </tr>
 
 {% endmacro %}
 
+
 {# <pre>{{garfile | dump(2)}}</pre> #}
-<table class="govuk-table">
+<table class="govuk-table" id="upt-responses">
   <thead class="govuk-table__head">
     <tr class="govuk-table__row">
       <th class="govuk-table__header" style="width:20%;">Status</th>
       <th class="govuk-table__header">Passenger</th>
       <th class="govuk-table__header">Permission to Travel</th>
     </tr>
   </thead>
+  {% if statusCheckComplete %}
+      <tr class="govuk-table__row">
+      <td class="govuk-table__cell"></td>
+      <td class="govuk-table__cell" colspan="2" style="opacity:.5;">This process is now complete.</td>
```

Comment:
This has bad contrast ration for accessibility
![Uploading Screenshot 2023-11-22 at 10.32.10.png…]()

Reply:
<img width="1440" alt="Screenshot 2023-11-22 at 10 32 10" src="https://github.com/UKHomeOffice/egar-public-site-ui/assets/123387957/994c55b6-5299-4538-9f2a-142a380499ff">